package.json

```diff
@@ -44,6 +45,8 @@
     "esbuild": "^0.19.8",
     "eslint": "^8.56.0",
     "eslint-config-prettier": "^9.1.0",
+    "istanbul-lib-coverage": "^3.2.2",
+    "istanbul-reports": "^3.1.7",
```

Comment:
why do we need this?

Reply:
@diegoalzate what do you mean? This are the packages required to produce the reports?

Reply:
is it because of a specific format you want? because the  --coverage flag from the latest jest does not require any other packages. My main question comes from the fact that these packages are not references anywhere so it is hard to tell where they are being used.

Reply:
okey I removed all istanbul dependencies and it still works. However, I had to remove them manually from the lock file.